tap_google_sheets/transform.py

```diff
@@ -145,37 +145,56 @@ def transform_sheet_boolean_data(value, unformatted_value, sheet_title, col_name
         return col_val
 
 # transform decimal values in the sheet
-def transform_sheet_decimal_data(value, sheet_title, col_name, col_letter, row_num, col_type):
+def transform_sheet_decimal_data(value_type, formatted_value, unformatted_value, sheet_title, col_name, col_letter, row_num, col_type):
```

Comment:
Add code function and code comments, explain if conditions and try catch

Reply:
Added function comment.